app/src/components/market/sections/market_bond/market_bond.tsx

```diff
@@ -132,15 +135,17 @@ const MarketBondWrapper: React.FC<Props> = (props: Props) => {
       })
 
       await fetchGraphMarketMakerData()
-
+      setBondOutcomeDisplay('0')
```

Comment:
@gem1no GJ! It works i would just add one suggestion since you are just changing the valueToDisplay if you after transaction you click bond it still has that values saved since you just changed the display value
![Screenshot 2021-05-20 at 12 39 50](https://user-images.githubusercontent.com/33226956/118965021-796f3280-b968-11eb-9c53-cf46ada1f37c.png)
So i would suggest to add setBondOutcomeSelected to Zero as well next to this if it doesn't break something